### ISSUE TITLE
2.0.0 Less than ideal layout on some resolutions, Patch Locations not shown

### ISSUE THREAD

**Issue by bboc** [christofmuc/KnobKraft-orm#221]
You can see from the screenshot, there's a lot of blank space around the categories, which squeezes  the actual content (the patches) very tightly at the bottom. The sliders for X and Y are not visible, either. 

One way to solve this is to put the sliders, search bar and dropdown into one row (either at the top or at the bottom), then the filters and categories can expand over the full with of the layout. 

<img width="696" alt="image" src="https://user-images.githubusercontent.com/520442/219135912-e5d7a72d-34db-489e-b351-c6f423074a1a.png">

**Comment by christofmuc** [comment]
Agree. Did you try to reduce the zoom via the View menu? The font is relatively big. What is your resolution here? 

I have started working on the second monitor feature to test layouting for small screens, which needs a bit of a different UX than the large desktop monitor I normally use.

**Comment by bboc** [comment]
My screen had somehow switched to 1200x800.  When I reduce the zoom, it's better, but there's plenty of space unused.
